spec/controllers/admin/users_controller_spec.rb

```diff
@@ -18,6 +18,18 @@
       expect(response).to render_template :index
     end
   end
+  describe 'PATCH #togle_confirmation' do
+    it 'confirms user' do
+      patch :toggle_confirmation, id: user.id, user: { to_confirm: 'true' }
```

Comment:
This makes sense only if we make sure that user is not already confirmed. `user` is already confirmed, so we can't really tell if this is working properly.
So before this action, I would create a new user, make sure confirmed_at is nil, and then execute the action.